### ISSUE TITLE
i followed the steps but there is a problem here : this.APP_ASSETS

### ISSUE THREAD

**Issue by zarga86** [mgechev/angular-seed#1830]
this.APP_ASSETS = [
      ...this.APP_ASSETS,
      // {src: `${this.APP_SRC}/your-path-to-lib/libs/jquery-ui.js`, inject: true, vendor: false}
      // {src: `${this.CSS_SRC}/path-to-lib/test-lib.css`, inject: true, vendor: false},
       { src: `${this.APP_SRC}/libs/js/scoll-to-top.js`, inject: true, vendor: false},
       // { src: `${this.APP_SRC}/libs/masterslider/jquery.easing.min.js`, inject: true, vendor: false },
       // { src: `${this.APP_SRC}/libs/masterslider/masterslider.min.js`, inject: true, vendor: false },
       // { src: `${this.APP_SRC}../../assets/js/script.js`, inject: true, vendor: false},
      //  { src: `${this.APP_SRC}/libs/masterslider/style/masterslider.css`, inject: true, vendor: false },
       // { src: `${this.APP_SRC}/libs/masterslider/ms-lightbox.css`, inject: true, vendor: false },
       // { src: `${this.APP_SRC}/libs/masterslider/skins/default/style.css`, inject: true, vendor: false },
       // { src: `${this.APP_SRC}/libs/masterslider/style.css`, inject: true, vendor: false }
    ];
this function didn't call to the Library local as it should do .
 my folder libs where should i put it . 
scoll-to-top.js : the same problem when i called to the function did not work but when i put an alert test outside the function it works 

could you help me to solve this problems thank you .  

![capture d ecran 2017-02-22 a 18 02 06](https://cloud.githubusercontent.com/assets/25960782/23222825/67548f40-f929-11e6-9e53-bdb91bdbed19.png)
![capture d ecran 2017-02-22 a 18 03 04](https://cloud.githubusercontent.com/assets/25960782/23222827/67bc2c5e-f929-11e6-955e-5a9f3a6ad05f.png)
![capture d ecran 2017-02-22 a 18 02 27](https://cloud.githubusercontent.com/assets/25960782/23222826/676bbfb2-f929-11e6-8740-2fbace563dcd.png)
![capture d ecran 2017-02-22 a 18 03 28](https://cloud.githubusercontent.com/assets/25960782/23222865/83b6b212-f929-11e6-8e42-900af832c102.png)

**Comment by Karasuni** [comment]
I can't comprehend your issue. I see that there is no problem with APP_ASSETS loading your `scroll-to-top.js`. Could you rephrase your issue and perhaps provide a fork that highlights the issue with **angular-seed**?

**Comment by zarga86** [comment]
when i called  scroll-to-top.js it didn't work it appear the page bt the fuction on it did't work .i other side master slide  in the local it work but when i integrated it on the angular seed the same issue didn't work the problem in the js pages all the css work normally  
![capture d ecran 2017-02-23 a 09 19 04](https://cloud.githubusercontent.com/assets/25960782/23251053/0caf4d04-f9ac-11e6-9d14-f49343816dee.png)
![capture d ecran 2017-02-23 a 09 21 23](https://cloud.githubusercontent.com/assets/25960782/23251054/0cb3291a-f9ac-11e6-9750-1dd6f74bca26.png)
![capture d ecran 2017-02-23 a 09 39 32](https://cloud.githubusercontent.com/assets/25960782/23251055/0cb77eac-f9ac-11e6-91ad-a983c305977a.png)

**Comment by Karasuni** [comment]
I recommend you provide a code example in the form of a fiddle or repository.

**Comment by zarga86** [comment]
this is the arborescence of my work including the libs files

![capture d ecran 2017-02-23 a 11 40 34](https://cloud.githubusercontent.com/assets/25960782/23255714/e926db34-f9bc-11e6-8bcb-f3ae4c2fd9c2.png)

and you found bellow the file witch contain the project.config and the html.could you tell me where is the problem please

[angular.zip](https://github.com/mgechev/angular-seed/files/796217/angular.zip)

**Comment by Karasuni** [comment]
That's not a working code sample. Create a fiddle or a repository with your specific issue.

However I believe that your issue is not a bug in angular-seed but rather the incorrect use of the third party libraries. I suggest creating a small page on your own using that library using pure HTML/JS - this is ideal in a fiddle. Make sure all your dependencies and required packages are loaded and use your browser's built-in debugger.

I can't help you any further at this point.
